.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
 Please contribute feedback on the overall design of WebAssembly, issues about
 the textual specification, or typos from the docs section of webassembly.org.
 
-For browser issues or tooling bugs, see links at http://webassembly.org/community/feedback/.
\ No newline at end of file
+For browser issues or tooling bugs, see links at https://github.com/WebAssembly/website/blob/master/community/feedback.md.
```

Comment:
> I don't believe the CG has been adequately consulted on this 'roadmap', that there is a good chance that a majority of the members would object to significant elements of the roadmap. For example the roadmap includes the ending of the CG and a transfer of power from the CG members to a W3C working group which would be dictated by the W3C and their customers at the exclusion of the CG and I for one could not possibly vote for such a change. There also appear to be significant areas of the design that have not been adequately explored, and I believe outright ignored, and it would appear that a change in the leadership of the group will be needed to complete a good design. Pointing 'contributors' to guidelines including a disputed roadmap and focusing on 'web developer' feedback when this project is desperate for guidance from language developers and help completing the design comes across as a statement of contempt as if 'this CG has the design under control thank you, here's your hat, please enter the logo contest on the way out' and I can not possibly support this tone.
>
> I also question the standing of some of the people claiming to be Chairs. I apologize in advance if I just missed the vote announcements but could the standing of the Chairs please be clarified?

@wllang the website was discussed in this repository ([design]https://github.com/WebAssembly/design/)) as well as in its own repository ([website](https://github.com/WebAssembly/website)). This issue is merely updating the contribution guidelines to reflect the consensus that was previously reached.

You seem to be disagreeing with that consensus. That's fine, but this issue isn't the right place. Regardless of what consensus is reached, the contributions should link to the roadmap.

The right place to object to the roadmap is on the [website](https://github.com/WebAssembly/website) repository.